mpu6050/am335x-bone-common.dtsi

```diff
@@ -67,6 +67,84 @@
 	};
 };
 
+&am33xx_pinmux {
```

Comment:
Please tell, why so massive changes in dtsi?

Reply:
I use the kernel version 4.14, please see https://github.com/beagleboard/linux/commit/82d75afcb718dc8fe9fcd71959bff7878df53076

Reply:
And it lead to issues to be fixed in "Remove wrong copy/past from am335x-bone-common.dtsi"

Anyway, if some commit is not yet merged, it is needed to rebase and modify it instead of adding a patch on top.